CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.25.0
+
+* Add functions to the built-in "sass:math" module.
```

Comment:
List the functions here, and provide brief descriptions of them.

Reply:
Done.